Code/GraphMol/SubstructLibrary/substructLibraryTest.cpp

```diff
@@ -692,38 +692,39 @@ void testSegFaultInHolder() {
 void testTautomerQueries() {
   BOOST_LOG(rdErrorLog) << "-------------------------------------" << std::endl;
   BOOST_LOG(rdErrorLog) << "   testTautomerQueries" << std::endl;
-    
+
   boost::shared_ptr<CachedTrustedSmilesMolHolder> mols1(
-							new CachedTrustedSmilesMolHolder());
+      new CachedTrustedSmilesMolHolder());
```

Comment:
maybe: 

    auto mols1 = boost::make_shared<CachedTrustedSmilesMolHolder>();

Reply:
It could be worh doing that at some point, but refactoring the tests wasn't the goal of the PR